person.rb

```diff
@@ -7,15 +10,22 @@ def initialize(age, name = 'Unknown', parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    super()
   end
 
   def of_age?
     @age >= 18
   end
 
+  def correct_name
+    @name
+  end
+
   private :of_age?
 
   def can_use_services?
     of_age?
   end
+
```

Comment:
- [ ] Again, you have required the rental class in the person class using `require_relative`. Then you should create the custom/helper method that will be invoked when a Rental is being created upon a single-person record. For example:

```ruby
  def add_rental(book, date)
    Rental.new(date, book, self)
  end
```